sdk/eventhub/event-hubs/migrationguide.md

```diff
@@ -38,6 +38,10 @@ We've also merged the functionality from `EventProcessorHost` from the `@azure/e
 `EventHubConsumerClient` in the `@azure/event-hubs` package, allowing `EventHubConsumerClient` to be the single
 point of entry for receiving of any type (from single partition, all partitions, or with load balancing and checkpointing features) within Event Hubs.
 
+Important note on checkpoints: `EventHubConsumerClient` does not support legacy checkpoint data.
+In order to meet the goals set for unifying checkpoint data across languages for the new set of Event Hubs libraries and to make
```

Comment:
This doesnt tell me why the unification was needed across languages. How about this:

> In order to support interopability between different programming languages, a unifying checkpoint format was needed. This and the need to support improvements to the algorithm used for ............ breaking changes were necessary.